src/telemetry/Manager.h

```diff
@@ -50,6 +59,112 @@ class Manager
 	 */
 	virtual void InitPostScript();
 
+	enum class MetricType
```

Comment:
Here and below: everything that's public in a header, should be having Doxygen comments (types, methods, public instance variables, etc.), including documenting parameters and return types.

Reply:
I think I covered these now.